.github/workflows/ci.yml

```diff
@@ -16,12 +16,14 @@ concurrency:
   cancel-in-progress: true
 
 env:
+  PYTHONDEVMODE: 1
   PIP_BREAK_SYSTEM_PACKAGES: 1
   PIP_ONLY_BINARY: numpy
   FORCE_COLOR: 3
   PYTEST_TIMEOUT: 300
   # For cmake:
   VERBOSE: 1
+  CMAKE_BUILD_PARALLEL_LEVEL: 2
```

Comment:
I'd rather use ninja, which computes this automatically, since it's not always 2 on all runners. Currently the `--prefix` runs use ninja (and I haven't solved Windows runs with Ninja, so those are MSVC). I wouldn't set this globally.